### PR TITLE
Feature: Add support for Hybrik as a service provider

### PR DESCRIPTION
**Pull request by rajaksc**
Initial support for using Hybrik as a service provider

### CODE REVIEW

File: provider/hybrik/hybrik.go [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+
+	if preset.Video.Width != "" {
+		var presetWidth int
+		presetWidth, err = strconv.Atoi(preset.Video.Width)
+		if err != nil {
+			return "", ErrVideoWidthNan
+		}
+		videoWidth = &presetWidth
+	} else {
+		videoWidth = nil
```
Comment (fsouza) [inline provider/hybrik/hybrik.go]: Same as before: ``videoWidth`` is already nil, so you can remove this ``else`` block.

File: provider/hybrik/hybrik.go [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+
+	if preset.Video.Height != "" {
+		var presetHeight int
+		presetHeight, err = strconv.Atoi(preset.Video.Height)
+		if err != nil {
+			return "", ErrVideoHeightNan
+		}
+		videoHeight = &presetHeight
+	} else {
+		videoHeight = nil
```
Comment (fsouza) [inline provider/hybrik/hybrik.go]: Again :D

File: provider/hybrik/hybrik.go [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+		videoProfile = ""
+		videoLevel = ""
+	}
+
+	p := hwrapper.Preset{
+		Key:         preset.Name,
+		Name:        preset.Name,
+		Description: preset.Description,
+		Kind:        "transcode",
+		Path:        "video-transcoding-api-presets",
```
Comment (fsouza) [inline provider/hybrik/hybrik.go]: Out of curiosity, what's the preset path? Should we make it configurable?

Comment (rajaksc) [inline provider/hybrik/hybrik.go]: Presets can be organized into folders, and the path is essentially the virtual folder path for the preset. It's purely for organizational purposes. In our web interface, you'll be able to see these presets and they'll be in a folder called "video-transcoding-api-presets". You can also query for presets by path via our API.

We'll expose this via an optional env variable.

File: config/config.go [hunk 4]
```diff
@@ -83,6 +83,7 @@ type Hybrik struct {
 	AuthKey        string `envconfig:"HYBRIK_AUTH_KEY"`
 	AuthSecret     string `envconfig:"HYBRIK_AUTH_SECRET"`
 	Destination    string `envconfig:"HYBRIK_DESTINATION"`
+	PresetPath     string `envconfig:"HYBRIK_PRESET_PATH"`
```
Comment (fsouza) [inline config/config.go]: You can add the struct tag ``default:"transcoding-api-presets"`` and drop [this if](https://github.com/NYTimes/video-transcoding-api/pull/185/files/b3e136a57be375baaccba9734867a9edcc153211..0641150ac2fc07ef3b4176602b500ed04a7ee061#diff-87bbf9707c283d056c8721f73fa58e78R419).

Comment (rajaksc) [inline config/config.go]: Updated. Good point, thanks!

**Pull request merged**